### DISCUSSION TITLE
开源共同体的治理模型 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#111] [Comments]
# 开源共同体的治理模型 | 夜天之书

随着越来越多新的要素进入开源领域，如何建立一个高效的开源共同体，如何维护一个富有价值的开源共同体，逐渐成为每个参与者或多或少关注的问题。 Apache 软件基金会为每个项目提供了基础的治理原则，并在项目孵化到顶级项目的过程中通过孵化器导师的帮助建立起开源项目的治理模型，但是这一模型对每个具体项目在特定时期未必是最优的。同时，其他不在 Apache 软件基金会治理的项目，虽然拥有灵活设计治理模型的自

https://www.tisonkun.org/2022/01/06/open-source-governance/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
从 X 上反射回来, 再次通读, 
直觉上还是应了那句话: 能力越大责任越大,
无论自由还是开源软件社区, 又或是标准的企业组织,
嘦是组织, 就必然包含了一些相通的矛盾以及决策;

但是, 从组织规律上来说, 肯定是国家>企业>社区,
为什么?
能够承担后果的空间潜力不同而已;

社区, 一但有任何不满, 可以立即用脚投票走人,
所以, 发展时间够长, 有足够社会影响力的社区, 核心吸引力早已不是什么技术追求了;
就象国家的凝聚力形成类似;

所以, 开源治理, 最需要的不是 0->1 的初始阶段, 也不是 100->无穷大的线性发展阶段,
而是 1->100 无序扩张阶段, 人心浮动, 又都有足以撕裂社区的影响力时...

这时利益不足以左右行为, 只有真诚和勤劳, 类似 POW 原则, 用数量积累出威望来...
这已经比资本加身, 用外边来兑换为影响力要公平的多.

**Reply by tisonkun** [reply]
听起来很有道理，让我看看做一些具体的工作来验证 ...
